test/contract/generatepin/testGeneratePin.js

```diff
@@ -0,0 +1,110 @@
+        it('Returns HTTP 400 status', (done) => {
+            request(pinServiceUrl)
+                .get('')
+                .query({phoneNumber: '+$447701111111'})
+                .set('Session-Id', VALID_SESSION_ID)
+                .expect(400)
+                .end((err, res) => {
+                    if (err) {
+                        console.log('error raised: ', err);
+                    }
+                    expect(err).to.be.equal(null);
+                    expect(res.text).to.equal('');
+                    done();
+                });
+        });
+    });
+
+    describe('Missing Session-Id which should produce a 400 Bad Request', () => {
+        it('Returns HTTP 400 status', (done) => {
+            request(pinServiceUrl)
+                .get('')
+                .query({phoneNumber: `${VALID_UK_LOCAL_TEST_NUMBER}`})
+                .expect(400)
+                .end((err, res) => {
+                    if (err) {
+                        console.log('error raised: ', err);
+                    }
+                    expect(err).to.be.equal(null);
+                    done();
+                });
+        });
+    });
+
+    describe('Valid International Number', () => {
+        it('Returns HTTP 200 status and pin number', (done) => {
+            request(pinServiceUrl)
+                .get('')
+                .query({phoneNumber: `${VALID_INTERNATIONAL_TEST_NUMBER}`})
+                .set('Session-Id', VALID_SESSION_ID)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) {
+                        console.log('error raised: ', err);
+                    }
+                    expect(err).to.be.equal(null);
+                    expect(res.header).to.have.property('content-length').eq(VALID_PIN_CONTENT_LENGTH);
+                    expect(res.text).is.not.equal(null);
+                    done();
+                });
+        });
+    });
+
+    describe('Valid UK With 44 Prefix Number', () => {
+        it('Returns HTTP 200 status and pin number', (done) => {
+            request(pinServiceUrl)
+                .get('')
+                .query({phoneNumber: `${VALID_UK_WITH_PREFIX_TEST_NUMBER}`})
+                .set('Session-Id', VALID_SESSION_ID)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) {
+                        console.log('error raised: ', err);
+                    }
+                    expect(err).to.be.equal(null);
+                    expect(res.header).to.have.property('content-length').eq(VALID_PIN_CONTENT_LENGTH);
+                    expect(res.text).is.not.equal(null);
+                    done();
+                });
+        });
+    });
+
+   describe('Valid UK Local Number', () => {
+        it('Returns HTTP 200 status and pin number', (done) => {
+            request(pinServiceUrl)
+                .get('')
+                .query({phoneNumber: `${VALID_UK_LOCAL_TEST_NUMBER}`})
```

Comment:
`VALID_UK_LOCAL_TEST_NUMBER ` doesn't ned to be in a template string

Reply:
Yes, updated.